perceval/backends/github.py

```diff
@@ -90,8 +107,20 @@ def fetch(self, from_date=DEFAULT_DATETIME):
             self._flush_cache_queue()
             issues = json.loads(raw_issues)
             for issue in issues:
+                self._add_users(issue)
                 yield issue
 
+    def _add_users_from_cache(self, issue):
```

Comment:
Same comment that as above. Do not modify the given object inside the method. It makes difficult to reuse it.